lib/Agrammon/Documentation.pm6

```diff
@@ -0,0 +1,111 @@
+use Agrammon::Model;
+
+#| Create LaTeX document of Model
+sub prepare-model( Agrammon::Model $model, :%technical! ) is export {
+
+    my @sections;
+    for $model.evaluation-order.reverse -> $module {
+
+        my @inputs;
+        for $module.input -> $input (:$name, :$description, *%) {
+            @inputs.push( %( :name(latex-escape($input.name)), :description(latex-escape($input.description)) ) );
+        }
+
+        my @outputs;
+        for $module.output -> $output (:$name, :$formula, :$description, *%) {
+            @outputs.push( %(:name(latex-escape($output.name)), :code($output.code), :$formula, :description(latex-escape($output.description)) ) );
+        }
+
+        my $tax = latex-escape($module.taxonomy);
+
+        @sections.push( %(
+            :module($tax), :title(Q:s"\subsection{$tax}"), :description(latex-escape($module.description)),
+            :@inputs, :@outputs
+        ));
+
+    }
+    return @sections;
+}
+
+sub create-latex( Str $model-name, Agrammon::Model $model, :%technical! ) is export {
+    my @sections = prepare-model( $model, :%technical );
+
+    say Qs:to/LATEX/;
```

Comment:
It's probably better design (and a bit faster) to have this build up a string and return it, then `say` the result. (Easy way: make an array of pieces, and then join it at the end.)

Reply:
Can even join them with `\n`.

Reply:
Indeed. Done ...